### ISSUE TITLE
[BUG] Layout height problem when using fixed navbar with no border

### ISSUE THREAD

**Issue by Shidersz** [ColorlibHQ/AdminLTE#3558]
**Describe the bug**
There is a `1px` overflow on the calculated layout height when using the next combination:
- Fixed navbar
- Fixed sidebar
- No border on the navbar

**To Reproduce**
1. Go to https://adminlte.io/themes/v3/pages/layout/fixed-topnav.html
2. Use the right control sidebar to enable **No border** on the header options and **Fixed** on the sidebar options.
3. See error, a vertical scroll will appear.

**Expected behavior**
There should not be a vertical scroll, as when the navbar has the border (uncheck **No Border** option).

**Screenshots**

**ISSUE**: Combination of fixed navbar and sidebar with no border on the navbar. You can see the vertical scroll at the right.

![Screenshot from 2021-04-02 13-26-11](https://user-images.githubusercontent.com/63609705/113434277-09f0b400-93b7-11eb-884d-87e8d40f08b1.png)

**NO ISSUE**: Combination of fixed navbar and sidebar with the default border on the navbar. There is no vertical scroll at the right.

![Screenshot_2021-04-02 AdminLTE 3 Fixed Navbar Layout](https://user-images.githubusercontent.com/63609705/113433967-88008b00-93b6-11eb-90ad-9e9be7c951f1.png)

**Environment (please complete the following information):**
- AdminLTE Version: v3.1.0
- Operating System: Not relevant
- Browser (Version): Not relevant

**Additional notes:**

When the issue appears, you can fix the problem from the browser's inspector if you change the next style applied to the `.content-wrapper` element:

```css
.layout-navbar-fixed .wrapper .content-wrapper {
    margin-top: calc(3.5rem + 1px);
}
```

To next one:

```css
.layout-navbar-fixed .wrapper .content-wrapper {
    margin-top: calc(3.5rem);
}
```

**Comment by danny007in** [comment]
can edit the image with red rectangle - tools which help u https://getsharex.com/downloads/

**Comment by REJack** [comment]
Fixed with 11c2a74. 😄
